app/src/main/java/org/wikipedia/nearby/NearbyFragment.kt

```diff
@@ -172,6 +186,41 @@ class NearbyFragment : Fragment(), LinkPreviewDialog.Callback {
         }
     }
 
+    private fun setUpSymbolManagerWithClustering(mapboxMap: MapboxMap, style: Style) {
+        val clusterColorLayers = arrayOf(
+            androidx.core.util.Pair(
+                0, ContextCompat.getColor(requireActivity(), ResourceUtil.getThemedAttributeId(requireContext(), R.attr.success_color))
+            )
+        )
+        val clusterOptions = ClusterOptions()
+            .withClusterRadius(60)
+            .withColorLevels(clusterColorLayers)
+            .withTextSize(literal(12f))
+            .withTextField(Expression.toString(get(POINT_COUNT)))
+            .withTextColor(Expression.color(fetchAttributeId(R.attr.paper_color)))
+        symbolManager = SymbolManager(binding.mapView, mapboxMap, style, null, null, clusterOptions)
+
+        // Clustering with SymbolManager doesn't expose a few style specifications we need.
+        // Accessing the styles in a fail-safe manner
+        try {
+            style.getLayer(CLUSTER_TEXT_LAYER_ID)?.apply {
+                this.setProperties(
+                    textFont(MARKER_FONT_STACK),
+                    textIgnorePlacement(true),
+                    textAllowOverlap(true)
+                )
+            }
+            style.getLayer(CLUSTER_CIRCLE_LAYER_ID)?.apply {
+                this.setProperties(
+                    circleStrokeColor(fetchAttributeId(R.attr.paper_color)),
```

Comment:
Same as above. We should use `ResourceUtil.getThemedColor(requireContext(), R.attr.paper_color)` instead.